qiskit_nature/second_q/properties/angular_momentum.py

```diff
@@ -161,10 +157,11 @@ def second_q_ops(self) -> dict[str, FermionicOp]:
         h_1 = x_h1 + y_h1 + z_h1
         h_2 = x_h2 + y_h2 + z_h2
 
-        h1_ints = OneBodyElectronicIntegrals(ElectronicBasis.SO, h_1)
-        h2_ints = TwoBodyElectronicIntegrals(ElectronicBasis.SO, h_2)
+        tensor = PolynomialTensor(
+            {"+-": h_1, "++--": _chem_to_phys(h_2)}, register_length=self._num_spin_orbitals
+        )
 
-        op = (h1_ints.to_second_q_op() + h2_ints.to_second_q_op()).simplify()
+        op = FermionicOp.from_polynomial_tensor(tensor).simplify()
```

Comment:
Especially `simplify` is one method I would like to avoid of making an in-place operation, since it is a lossy conversion.
If you think this is really needed, we could have an in-place and an out-of-place equivalent method.

Reply:
I have opened #860 to track this :+1: